presto-native-execution/scripts/setup-macos.sh

```diff
@@ -16,25 +16,31 @@ set -eufx -o pipefail
 # Run the velox setup script first.
 source "$(dirname "${BASH_SOURCE}")/../velox/scripts/setup-macos.sh"
 
-MACOS_DEPS="${MACOS_DEPS} bison gperf"
 export FB_OS_VERSION=v2024.04.01.00
 
-export PATH=$(brew --prefix bison)/bin:$PATH
```

Comment:
None of the prestissimo/velox deps require bison? 
On Silicon Macs this will be a problem going forward but is likely set later by the users. But removing this we don't expect issues?

Reply:
This is a temporary assignment on the current shell. I am thinking there should not be any issues since we don't run this script on every build and things build fine.

Reply:
Also, bison is installed in the Velox setup script.